templates/adp-backend-template-dotnet/skeleton/src/${{ values.dotnet_solution_name }}.Api/${{ values.dotnet_solution_name }}.Api.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<Version>0.1.1</Version>
+		<Nullable>enable</Nullable>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+		<DockerfileContext>.</DockerfileContext>
+		<DockerfileFile>..\..\Dockerfile</DockerfileFile>
+		<UserSecretsId>79971c91-7a80-4368-99e9-a67eb70d026b</UserSecretsId>
```

Comment:
This id shouldnt be hard coded, it will cause all projects scaffolded from this to share a common secrets file. It should be randomly generated each time a project is scaffolded, or omitted and left to the user to generate on demand